services/frontend-v3/src/App.css

```diff
@@ -268,8 +274,12 @@ a.ant-btn.ant-btn-link.ant-btn-block:focus {
 }
 
 button.ant-btn.ant-btn-link:focus {
-  background-color: #08bab462;
-  color: #001c1a;
+  background-color: #415a5a;
+
+  box-shadow: 0px 18px #415a5a, 0px 7px #415a5a;
+  padding-top: 20px;
+  padding-bottom: 0px;
+  border-radius: 20px;
```

Comment:
I'd suggest lowering this border radius value. The individual boxes making up the dropdown highlight start to show through at 20px and it looks kinda weird.

![css radius](https://user-images.githubusercontent.com/30152653/85132464-d61d1700-b206-11ea-8d1c-59f89ca742c0.png)


```suggestion
  border-radius: 10px;
```

Reply:
Wow, I dont know how I didnt see the curved sides with 20px radius- good catch thank you Trevor!